seyik/services/postService.js

```diff
@@ -0,0 +1,41 @@
+//service/postService.js
+
+const postDao = require("../models/postDao");
+
+const createPost = async (title, content, userId) => {
+  const createPost = await postDao.createPost(title, content, userId);
+
+  return createPost;
+};
```

Comment:
변수 사용을 안 하면 굳이 선언하지 않고 바로 return 해주면 됩니다.
```suggestion
const createPost = async (title, content, userId) => {
  return await postDao.createPost(title, content, userId);
};
```